src/experimental-bots/filler/fillerMultithreaded.ts

```diff
@@ -2018,13 +2082,31 @@ export class FillerMultithreaded {
 		}
 	}
 
+	/**
+	 * Gives filler reward estimate
+	 *
+	 * @param taker
+	 * @param quoteAssetAmount
+	 */
+	protected calculateFillerRewardEstimate(
+		feeTier: FeeTier,
+		quoteAssetAmount: BN
+	) {
+		const takerFee = quoteAssetAmount
+			.muln(feeTier.feeNumerator)
+			.divn(feeTier.feeDenominator);
+		const fillerReward = BN.min(new BN(10_000), takerFee.muln(10).divn(100));
```

Comment:
nit: can just divide taker fee by 10?

Reply:
was being safe for precision's sake. but will change to divn